src/Cake.Svn/SvnAliases.Delete.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Svn.Delete;
+
+namespace Cake.Svn
+{
+    public static partial class SvnAliases
+    {
+        /// <summary>
+        /// Delete a file or directory from Subversion.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="fileOrDirectory">The file or directory to delete.</param>
+        /// <returns>True if the command were successful. Otherwise False will be returned.</returns>
+        /// <example>
+        /// <para>Add a file or a directory to Subversion.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var fileRemoved = SvnDelete(
+        ///         @"C:\project\src\newfile.cs");
+        ///
+        ///     var directoryRemoved = SvnDelete(
+        ///         @"C:\project\src\newfolder\");
+        ///
+        ///     Verbose("File removed: {0}", fileRemoved);
+        ///     Verbose("Directory removed: {0}", directoryRemoved);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Delete")]
+        [CakeNamespaceImport("Cake.Svn.Delete")]
+        public static bool SvnDelete(this ICakeContext context, string fileOrDirectory)
```

Comment:
Cake has [FilePath](https://cakebuild.net/api/Cake.Core.IO/FilePath/) and [DirectoryPath](https://cakebuild.net/api/Cake.Core.IO/DirectoryPath/) types which should be used for passing file or directory paths. Since file and paths are supported this will require an overload of the aliases.

For passing multiple files or directories, there is also [FilePathCollection](https://cakebuild.net/api/Cake.Core.IO/FilePathCollection/) and [DirectoryPathCollection](https://cakebuild.net/api/Cake.Core.IO/DirectoryPathCollection), but I'm fine to add this in a later PR (need to also think about appropriate return value in this case).